src/VisualStudio/Core/Def/xlf/ServicesVSResources.zh-Hant.xlf

```diff
@@ -122,6 +177,11 @@
         <target state="translated">重設 Visual Studio 預設按鍵對應</target>
         <note />
       </trans-unit>
+      <trans-unit id="Review_problems_below_Click_finish_to_proceed_pull_up">
+        <source>Review problems below. Click Finish to proceed pull up.</source>
+        <target state="new">Review problems below. Click Finish to proceed pull up.</target>
```

Comment:
Someone needs to review this.  I don't think dialogs generally tell you the button you need to use.  Also 'click' is very mouse-centric.

Reply:
Sure, I will try to make these words more accurate.